src/main/resources/config/config.yaml

```diff
@@ -13,9 +13,10 @@ logging:
       threshold: ALL
       timeZone: UTC
       target: stdout
+      customFields:
+        label: "directdebit-connector"
       layout:
         type: json
```

Comment:
I don't think we use the layout at all, the Logstash Encoder logs in JSON by default right?

Reply:
you're right. i've removed the `layout` now.